spark-mongodb/src/main/scala/com/stratio/datasource/util/Config.scala

```diff
@@ -62,7 +62,7 @@ abstract class ConfigBuilder[Builder<:ConfigBuilder[Builder] ](
    */
   def build(): Config = new Config {
 
-    val properties = builder.properties
+    val properties = builder.properties.map(kv => kv.copy(_1 = kv._1.toLowerCase))
```

Comment:
This approach is more readable:

`val properties = builder.properties.map { case (k, v) => k.toLowerCase -> v }`